doc/source/index.rst

```diff
@@ -6,10 +6,23 @@
 Welcome to Language Model Playground's documentation!
 =====================================================
 
+Language Model Playground is based on Pytorch. It contains several models including RNN, LSTM, GRU, Attention,
```

Comment:
Put single sentence in the one line, only use multiline when passing length limit (80 characters)